SiteAssets/resources/pnp.js

```diff
@@ -127,7 +127,7 @@ var Util = /** @class */ (function () {
      * @param name The name of the url paramter to check
      */
     Util.urlParamExists = function (name) {
-        name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+        name = name.replace(/\\/g, "\\\\").replace(/\[/g, "\\[").replace(/\]/g, "\\]");
```

Comment:
Just want to make sure that we should be replacing back slashes with double backslashes, this part of the code: replace(/\\/g, "\\\\")

Reply:
The additional backslash is used to ensure that any backslashes in the original string are properly escaped, preventing them from being misinterpreted as escape characters.